lib/ansible/modules/network/checkpoint/checkpoint_session.py

```diff
@@ -104,7 +104,8 @@ def main():
             code, response = connection.send_request('/web_api/publish', payload)
         else:
             code, response = connection.send_request('/web_api/discard', payload)
-
+        if code != 200:
```

Comment:
also will never run because the higher scope if statement `if code == 200` has to be `True` to even get here

Reply:
same as above